sapi/fpm/fpm/fpm_status.c

```diff
@@ -430,7 +430,8 @@ int fpm_status_handle_request(void) /* {{{ */
 				"phpfpm_max_children_reached %u\n"
 				"# HELP phpfpm_slow_requests The number of requests that exceeded your 'request_slowlog_timeout' value.\n"
 				"# TYPE phpfpm_slow_requests counter\n"
-				"phpfpm_slow_requests %lu\n";
+				"phpfpm_slow_requests %lu\n"
+				"# EOF\n";
```

Comment:
Style nit: why not write that on a single line?
```suggestion
				"phpfpm_slow_requests %lu\n# EOF\n";
```

Reply:
No particular reason, I thought that the format was one significant word per line and moreover if a new metric will ever be added the diff change will not involve this line, but if you prefer I can make it inline

Reply:
Thanks! I'm fine with your reasoning. Let's wait on review by somebody actually involved with PHP-FPM.

Reply:
Honestly, I would do it line by line like this as well. Mimic the expected output and also for the reason @ste93cry mentions 👍